coral-hive/src/test/java/com/linkedin/coral/hive/hive2rel/parsetree/ParseTreeBuilderTest.java

```diff
@@ -88,6 +88,7 @@ public static void beforeClass() throws HiveException, IOException, MetaExceptio
         "SELECT * from foo left outer join bar on foo.a = bar.b",
         "SELECT * from foo right outer join bar on foo.a = bar.b",
         "SELECT * from foo full outer join bar on foo.a = bar.b",
+        "SELECT foo.*, bar.b from foo join bar on foo.a = bar.b",
```

Comment:
Please also add a test in `HiveToTrinoConverterTest` to make sure the conversion into Trino SQL is done right too.

Reply:
Added